src/main/steps/common/task-list/utils.ts

```diff
@@ -91,5 +91,15 @@ export const keepDetailsPrivateNav = (caseData: Partial<CaseWithId>, req: AppReq
 
 export const isCafcassServed = (caseData: Partial<CaseWithId>): boolean => caseData?.isCafcassServed === YesOrNo.YES;
 
-export const isCafcassCymruServed = (caseData: Partial<CaseWithId>): boolean =>
-  caseData?.isCafcassCymruServed === YesOrNo.YES;
+export const isCafcassCymruServed = (caseData: Partial<CaseWithId>): boolean => {
+  const lengthOfServedApplicationDetailsList = caseData.finalServedApplicationDetailsList?.length as number;
+  if (
+    lengthOfServedApplicationDetailsList >= 1 &&
+    caseData.finalServedApplicationDetailsList?.find(list =>
+      list.value.emailNotificationDetails?.find(i => i.value?.servedParty === 'Cafcass cymru')
```

Comment:
Can u pls chk if 'Cafcass cymru' is an enum or text?

Reply:
it is a string, but still changed to enum. In future for party specific document handle this enum can be helpful

Reply:
@sealDebolina , Pls check instead of string can the backend send id for Cafcass, Cafcass Cymru so UI need not to worry even if the text changes in the future for any other language.